src/Menu/SelectMenu.jsx

```diff
@@ -165,12 +166,15 @@ SelectMenu.propTypes = {
   children: PropTypes.node.isRequired,
   /** Specifies class name to append to the base element */
   className: PropTypes.string,
+  /** Specifies variant to use. */
+  variant: PropTypes.string,
 };
 
 SelectMenu.defaultProps = {
   defaultMessage: SELECT_MENU_DEFAULT_MESSAGE,
   isLink: false,
```

Comment:
We should deprecate `isLink` prop now as you can just pass `variant="link"` to get required styles. This is a good opportunity to use `withDeprecatedProps` HOC, see `Card` component for example where we deprecated `muted` prop.